packages/xo-server/src/vhd-merge.js

```diff
@@ -549,11 +546,21 @@ export class Vhd {
     await this._write(block.buffer, sectorsToBytes(blockAddr))
   }
 
-  async writeBlockSectors (block, beginSectorId, endSectorId, parentBitmap) {
+  async writeBlockSectors (
+    block,
+    beginSectorId,
+    endSectorId,
+    parentBitmap = null
```

Comment:
Why this change?

Reply:
it's for the write operation, it doesn't have a parent bitmap to merge

Reply:
You don't need to use `null` as the default value, simply test for `undefined`.